docs/developer/index.rst

```diff
@@ -15,3 +14,17 @@ Version number
 The version number is currently stored in 3 places::
 
     PIL/__init__.py _imaging.c setup.py
+
+
+Readthedocs
+~~~~~~~~~~~
+
+Make sure the default version for Readthedocs is the latest release version e.g. 3.0.0 not latest.
```

Comment:
"Readthedocs" -> "Read the Docs". Also as a developer, and as a collaborator, I've no idea where this version is.